zbus/src/blocking/proxy/mod.rs

```diff
@@ -527,3 +527,19 @@ mod tests {
         assert!(signal.args().unwrap().name() == well_known);
     }
 }
+
+/// This trait is implemented by all blocking proxies, which were
+/// generated with the [`dbus_proxy`](zbus::dbus_proxy) macro.
```

Comment:
Does this way of linking works? :thinking:

Reply:
![chrome_sl6iP9qlT1](https://github.com/dbus2/zbus/assets/12705416/6591eeac-87f2-43e3-9f72-1abc3bfb93ef)


Yes it does 🙃

Reply:
Nice. TIL :)